Chinese/YingJeouMa/YingJeouMa.md

```diff
@@ -1,3 +1,5 @@
 成為某國總統，禍國殃民。
 
-最喜歡的人有三個，一個是金葡蔥，一個是周沒清，另一個我不能說。
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.